full-service/src/service/decorated_types.rs

```diff
@@ -8,7 +8,9 @@ use crate::db::{
     txo::TxoDetails,
 };
 use chrono::{TimeZone, Utc};
-use mc_mobilecoind_json::data_types::{JsonTxOut, JsonTxOutMembershipElement};
+use mc_mobilecoind_json::data_types::{
```

Comment:
It seems odd to me that full-service depends on mobilecoind_json. Full-service is supposed to replace mobilecoind and mobilecoind_json, right? Maybe more importantly, its awkward that the api of this service is partly defined in a different repo.

Reply:
Yeah - the goal here was to make TxProposal cross-compatible with the wallets, so that wherever you built your TxProposal, you could submit it with a compatible wallet.